algo_es.go

```diff
@@ -69,6 +69,10 @@ func (h esAlg) Algorithm() Algorithm {
 	return h.alg
 }
 
+func (h esAlg) SignSize() int {
+	return h.privateKey.Curve.Params().BitSize / 8
```

Comment:
So sometimes it's useful to divide on 8, right?:)

Reply:
Yep)